pkg/upgrade/upgrade_suite_test.go

```diff
@@ -0,0 +1,124 @@
+package upgrade_test
+
+import (
+	"testing"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+)
+
+func TestUpgrade(t *testing.T) {
+	gomega.RegisterFailHandler(ginkgo.Fail)
+	ginkgo.RunSpecs(t, "Upgrade Suite")
+}
+
+// Declarations for Ginkgo DSL
```

Comment:
Are all these auto-generated ? Or can we pull them in dynamically or something. I'm not familiar enough with ginkgo.

Reply:
See: https://onsi.github.io/ginkgo/#avoiding-dot-imports
```
# *_suite_test.go one-time generated via:
ginkgo bootstrap --nodot
# with some minor cleanup/edits 
```